Avoid panic reading preservation actions/tasks

jraddaoui opened the pull request:
Transform NULL `started_at` dates to zero value dates on their Goa representations.

Refs https://github.com/artefactual-sdps/enduro/issues/875.

Diogenesoftoronto reviewed (approved):
LGTM. Remember to put [skip codecov] in your commit so that it doesn't break ci

jraddaoui committed:
Avoid panic reading preservation actions/tasks

Transform NULL `started_at` dates to zero value dates on their Goa representations.

[skip codecov]
